assets/src/edit-story/components/panels/animation/effectChooser.js

```diff
@@ -412,11 +385,51 @@ export default function EffectChooser({
               aria-disabled={disabledBackgroundEffects.includes(
                 PAN_MAPPING[DIRECTION.TOP_TO_BOTTOM]
               )}
-              active={activeEffectListIndex === 5}
+              active={activeEffectListIndex === 4}
             >
               <ContentWrapper>{__('Pan Down', 'web-stories')}</ContentWrapper>
               <PanTopAnimation>{__('Pan Down', 'web-stories')}</PanTopAnimation>
             </GridItem>
+            <GridItemHalfRow
+              aria-label={__('Zoom In Effect', 'web-stories')}
+              onClick={(event) =>
+                handleOnSelect(event, BACKGROUND_ANIMATION_EFFECTS.ZOOM.value, {
+                  animation: BACKGROUND_ANIMATION_EFFECTS.ZOOM.value,
+                  zoomDirection: SCALE_DIRECTION.SCALE_IN,
+                })
+              }
+              aria-disabled={disabledBackgroundEffects.includes(
+                getDirectionalEffect(
+                  BACKGROUND_ANIMATION_EFFECTS.ZOOM.value,
+                  SCALE_DIRECTION.SCALE_IN
+                )
+              )}
+              active={activeEffectListIndex === 5}
+            >
+              <ContentWrapper>{__('Zoom In', 'web-stories')}</ContentWrapper>
+              <ZoomInAnimation>{__('Zoom In', 'web-stories')}</ZoomInAnimation>
```

Comment:
For foreground objects it's called Scale In, for Background we want to keep Zoom correct?

Reply:
yup, that is correct!